ui/package.json

```diff
@@ -114,8 +114,8 @@
     "react-image-size": "^2.0.0",
     "react-intersection-observer": "^9.4.0",
     "react-oembed-container": "^1.0.1",
-    "react-router": "^6.3.0",
-    "react-router-dom": "^6.3.0",
+    "react-router": "^6.6.2",
```

Comment:
This changed the way relative links work, currently getting a bad redirect when entering a group where the previous recent channel sends me to '../../activity'

Reply:
Good catch; I spiked into updating our usage of relative paths, but since there are many it would be non-trivial. Something we should consider in the future. For now, downgraded back to 6.3.0 (the release before the breaking change).